scrapi/harvesters/scitech.py

```diff
@@ -55,20 +51,21 @@ def harvest(self, days_back=1):
                 'doc': etree.tostring(record),
                 'docID': record.xpath('dc:ostiId/node()', namespaces=self.namespaces)[0].decode('utf-8'),
             })
-            for record in self._fetch_records(days_back)
+            for record in self._fetch_records(start_date, end_date)
         ]
 
-    def _fetch_records(self, days_back):
+    def _fetch_records(self, start_date, end_date):
         page = 0
         morepages = True
-        to_date = datetime.date.today().strftime('%m/%d/%Y')
-        from_date = (datetime.date.today() - datetime.timedelta(days_back)).strftime('%m/%d/%Y')
+
+        start_date = start_date or date.today()
+        end_date = end_date or date.today() - timedelta(1)
 
         while morepages:
             resp = requests.get(self.base_url, params={
                 'page': page,
-                'EntryDateTo': to_date,
-                'EntryDateFrom': from_date,
+                'EntryDateTo': end_date.strftime('%m/%d/%Y'),
```

Comment:
If this ```strftime('%m/%d/%Y')``` pops up a lot, you may want to stick it into a function for readability.